hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/server/TestContainerServer.java

```diff
@@ -192,6 +195,9 @@ private static HddsDispatcher createDispatcher(DatanodeDetails dd, UUID scmId,
     conf.set(OZONE_METADATA_DIRS, TEST_DIR);
     VolumeSet volumeSet = new MutableVolumeSet(dd.getUuidString(), conf, null,
         StorageVolume.VolumeType.DATA_VOLUME, null);
+    Path tempDir = Files.createTempDirectory("");
```

Comment:
This tempDir should be explicitly deleted, other wise it will leak disk space on CI server, resulting out of space error at last. 
Same for change in TestSecureContainerServer.java.